src/profile/supporting_types/authorized_dapp_detailed.rs

```diff
@@ -23,6 +23,9 @@ pub struct AuthorizedDappDetailed {
     pub display_name: Option<DisplayName>,
 
     pub detailed_authorized_personas: DetailedAuthorizedPersonas,
+
+    #[serde(default)]
```

Comment:
@matiasbzurovski I think you can remove 

```
    Serialize,
    Deserialize,
``` 
from `AuthorizedDappDetailed`

and also all `#serde` ofc... It seems it is not needed! does that compile?

Reply:
right, that makes more sense based on our Slack discussion. Removed